### PR TITLE
refactor: concurrent table scan in "field columns"

### PR DESCRIPTION
**Pull request by crepererum**
Similar to #5647 and #5649.

### CODE REVIEW

**Comment by crepererum** [comment]
Oh now, this time I broke it :D Will have a look later.

**Comment by crepererum** [comment]
Fixed.

File: iox_query/src/frontend/influxrpc.rs [hunk 1]
```diff
@@ -1313,33 +1327,30 @@ fn table_chunk_stream<'a>(
         .buffered(CONCURRENT_TABLE_JOBS)
 }
 
-/// Create series set plans that fetch the data specified in rpc_predicate.
+/// Create plans that fetch the data specified in rpc_predicate.
```
Comment (alamb) [inline iox_query/src/frontend/influxrpc.rs:1330]:
```suggestion
/// Create plans that fetch the data specified in table_predicates.
///
/// table_predicates contains `(table_name, predicate_specialized_for_that_table)`
///
/// The plans are created in parallel as different `async` Tasks to reduce 
/// query latency due to planning
```

**Review by alamb — APPROVED** [review]
Looks very nice to me. I can't wait to see how query performance improves after this work

**Pull request merged**